src/bundle/Resources/public/scss/_alloyeditor-custom-styles.scss

```diff
@@ -20,3 +20,7 @@
         font-family: courier, monospace;
     }
 }
+ul.ae-listbox {
```

Comment:
Those styles are not related to custom styles. It will affect all `lists` in AE so I would move this to the `ezplatform-admin-ui/src/bundle/Resources/public/css/alloyeditor/alloyeditor-ez.css`
Also, I would change the selector to match AE selector: `.ae-ui .ae-dropdown .ae-listbox`

Reply:
@dew326 Please have a look now. I used and tested your proposition.